demo/d2l-filter-dropdown/templated-filter-dropdown-demo.js

```diff
@@ -0,0 +1,193 @@
+import '../../components/d2l-filter-dropdown/d2l-filter-dropdown.js';
+import '../../components/d2l-filter-dropdown/d2l-filter-dropdown-category.js';
+import '../../components/d2l-filter-dropdown/d2l-filter-dropdown-option.js';
+import { html, LitElement } from 'lit-element';
+
+/**
+ * An example of a templated filter-dropdown.
+**/
+class TemplatedFilterDropdownFilter extends LitElement {
+	static get properties() {
+		return {
+			_filters: { type: Array },
+			_totalSelected: { type: Number }
```

Comment:
If these aren't intended to be set by consumers (i.e. private), then you can add `attribute: false` to remove all the attribute binding stuff.